### ISSUE TITLE
1.1.9 broke our app

### ISSUE THREAD

**Issue by singpolyma** [KyleAMathews/react-spinkit#39]
Upgrading to 1.1.9 changed the item exported by this library from a directly-usable react component to an object with a `default` property.  Does it require a newer version of React now (should be reflected in dependencies?) or perhaps this is an interface change that warrants a major version number change?

We're on React 0.13.x, if that matters

**Comment by KyleAMathews** [comment]
Oh drat, with the change to ES2015, we switched the module style from commonjs. That was unnecessary and I just [switched it back](https://github.com/KyleAMathews/react-spinkit/commit/451ac333a607345eb2ebb32fca10cb3cd7d384a3) and released 1.1.10. Let me know if it works now.

Thanks for reporting the issue!

**Comment by zbigg** [comment]
Looks like 1.1.10 isn't built correctly. When i install it, i still see that `react-spinkit` exports ES6-like module with constructor in `default property`.

```
$ npm install
web4@1.0.0 /Users/zbigg/projects/tiq_backend/web
└── react-spinkit@1.1.10 
Object.defineProperty(exports, "__esModule", {
exports.default = Spinner;
```

Note, that i've checked-out `v1.1.10` and it builds correctly:

```
$ git checkout v1.1.10
Previous HEAD position was eedffc4... 1.1.9
HEAD is now at 9c19e73... 1.1.10
|1|x86_64| zbigg@antari:/Users/zbigg/projects/react-spinkit
$ npm run build
(...)
src/index.jsx -> dist/index.js
|1|x86_64| zbigg@antari:/Users/zbigg/projects/react-spinkit
$ grep exports dist/index.js 
module.exports = Spinner;
```

**Comment by KyleAMathews** [comment]
Oh boo. Some of my projects auto build on publishing and others don't... this one didn't and I forgot to build it. Just published 1.1.11. Hopefully that fixes things. Thanks for the report.
